robotbase/src/main/java/frc/robot/commands/intake/IntakeToShooterCommand.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Robot;
+
+public class IntakeToShooterCommand extends Command {
+
+  public IntakeToShooterCommand() {
+    addRequirements(Robot.intake);
+  }
+
+  @Override
+  public void initialize() {}
+
+  @Override
+  public void execute() {
+    Robot.intake.setAxisSpeed(
+        Constants.INTAKE.TOP_MOTOR_SPEED_PERCENT_OUTPUT,
+        Constants.INTAKE.BOTTOM_MOTOR_SPEED_PERCENT_OUTPUT);
+  }
+
+  @Override
+  public boolean isFinished() {
+    return false;
+  }
```

Comment:
@NolanCampbell2357 is this fine? This is what it was in the example from 2022. I can't think of any reliable way to check if we're done other than deadlining it with a wait command. If we do it based on the beam break, it'll most likely stop the motors sooner than we want.

Reply:
I was told that the wait command would be scheduled outside of this, in another command.